### PR TITLE
Add performance and longevity testing validation to the release template

### PR DESCRIPTION
**Pull request by travisbenedict**
Signed-off-by: Travis Benedict <benedtra@amazon.com>

### Description
Add steps to the release template for validating performance and longevity test results. These steps are not necessarily exhaustive but can serve as a starting point for our release process. Ultimately, these validations should be automated as our performance testing automation is built out more. 

More details on the testing setup can be found in this issue outlining the results of performance experiments that I conducted - https://github.com/opensearch-project/OpenSearch/issues/2461
 
### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/test_workflow/README.md [hunk 1]
```diff
@@ -99,7 +99,65 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
 
 ### Performance Tests
 
-TODO
+TODO: Add instructions on how run performance tests with `test.sh`
+
+
+
+#### How to identify regressions in performance tests
```
Comment (dblock) [inline src/test_workflow/README.md:106]: Add to TOC, cleanup capitalization to match other topics.

I would rename to "Identifying Regressions in Performance Tests"

File: src/test_workflow/README.md [hunk 2]
```diff
@@ -99,7 +99,65 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
...
+|Instance Type|Security|Expected Indexing Throughput Avg (req/s)|Expected Indexing Error Rate|Indexing StDev% Mean|Indexing MinMax% Diff|Expected Query Latency p90 (ms)|Expected Query Latency p99 (ms)|Expected Query Error Rate|Query StDev% Mean|Query MinMax% Diff|
+|---|---|---|---|---|---|---|---|---|---|---|
+|m5.xlarge|Enabled|30554|0|~5%|~12%|431|449|0|~10%|~23%|
+|m5.xlarge|Disabled|34472|0|~5%|~15%|418|444|0|~10%|~25%|
+|m6g.xlarge|Enabled|38625|0|~3%|~8%|497|512|0|~8%|~23|
+|m6g.xlarge|Disabled|45447|0|~2%|~3%|470|480|0|~5%|~15%|
+
+Note that performance regressions are based on decreased indexing throughput and/or increased query latency.
+
+Additionally, error rates on the order of 0.01% are acceptable, though higher ones may be cause for concern.
```
Comment (dblock) [inline src/test_workflow/README.md:133]: This should be a section of its own. What happens if the error rates are higher? What does one do?

File: src/test_workflow/README.md [hunk 3]
```diff
@@ -99,7 +99,65 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
...
+
+
+
+#### How to identify issues in longevity tests
```
Comment (dblock) [inline src/test_workflow/README.md:137]: There's zero information on what longevity tests are anywhere in these docs. To an uneducated reader it's impossible to understand what one does with it. Please provide context to all these things.

**Review by dblock — CHANGES_REQUESTED** [review]

File: src/test_workflow/README.md [hunk 4]
```diff
@@ -99,7 +99,65 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
...
-TODO
+TODO: Add instructions on how run performance tests with `test.sh`
+
+
+
+#### How to identify regressions in performance tests
+
+Disclaimer: the guidelines listed below were determined based on empirical testing using OpenSearch Benchmark. 
+These tests were run against OpenSearch 1.2 build #762 and used the nyc_taxis workload with 2 warmup and 3 test iterations. 
+The values listed below are **not** applicable to other configurations. More details on the test setup can be found here: https://github.com/opensearch-project/OpenSearch/issues/2461
```
[the remark this answers is not in the document]
Comment (travisbenedict) [reply src/test_workflow/README.md:110]: Good points. I'll rewrite this section

File: src/test_workflow/README.md [hunk 5]
```diff
@@ -3,13 +3,16 @@
...
+      - [Identifying Regressions in Nightly Performance Tests](#identifying-regressions-in-nightly-performance-tests)
+    - [Identifying Issues in Longevity Tests](#identifying-issues-in-longevity-tests)
 - [Testing in CI/CD](#testing-in-cicd)
-  - [Test Workflow (in development)](#test-workflow-in-development)
-  - [Component-Level Details](#component-level-details)
-    - [test-orchestrator pipeline](#test-orchestrator-pipeline)
-    - [integTest job](#integtest-job)
-    - [bwcTest job](#bwctest-job)
-    - [perfTest job](#perftest-job)
+    - [Test Workflow (in development)](#test-workflow-in-development)
```
Comment (travisbenedict) [inline src/test_workflow/README.md:10]: These got reformatted by my IDE. Will move them back

File: src/test_workflow/README.md [hunk 6]
```diff
@@ -99,7 +102,70 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
 
 ### Performance Tests
 
-TODO
+TODO: Add instructions for running performance tests with `test.sh`
+
+
+Performance tests from `test.sh` are executed using an internal service which automatically provisions hosts that run [OpenSearch Benchmark](https://github.com/opensearch-project/OpenSearch-Benchmark). Work to open source these internal features is being tracked [here](https://github.com/opensearch-project/opensearch-benchmark/issues/97).
```
[the remark this answers is not in the document]
Comment (travisbenedict) [reply src/test_workflow/README.md:108]: I like that a lot better. Thanks!

**Comment by travisbenedict** [comment]
Thanks for the help on this @dblock. I really appreciate the feedback

**Pull request merged**